client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1494,6 +1494,25 @@ Undocumented.prototype.jetpackThemeDetails = function( themeId, siteId, fn ) {
 	}, fn );
 };
 
+/**
+ * Install a theme from WordPress.org or WordPress.com on the given Jetpack site.
+ * Whether the theme is installed from .com or .org is controlled by the themeId string
+ * if it has a -wpcom suffix, .com is used.
+ *
+ * @param {String}    siteId   The site ID
+ * @param {String}    themeId  WordPress.com theme with -wpcom suffix, WordPress.org otherwise
+ * @param {Function}  fn       The callback function
+ * @returns {Promise} promise
+ */
+Undocumented.prototype.installThemeOnJetpack = function( siteId, themeId, fn ) {
```

Comment:
I haven't been adding the callback func in recent calls I've added to this module, since we always use the returned promise.

I don't see any harm in adding it, though.